services/nomad/build/build-rsyncd.nomad

```diff
@@ -0,0 +1,59 @@
+job "build-rsyncd" {
+  type = "service"
+  datacenters = ["VOID"]
+  namespace = "build"
+
+  group "rsync" {
+    count = 1
+    network {
+      mode = "host"
+    }
+
+    volume "root-pkgs" {
+      type = "host"
+      source = "root-pkgs"
+      read_only = false
+    }
+
+    task "rsyncd" {
+      driver = "docker"
+
+      vault {
+        policies = ["void-secrets-repomgmt"]
+      }
+
+      config {
+        image = "ghcr.io/void-linux/infra-rsync:202308" # TODO
+        volumes = [
+            "local/buildsync.conf:/etc/rsyncd.conf.d/buildsync.conf",
+        ]
+        network_mode = "host"
+      }
+
+      volume_mount {
+        volume = "root-pkgs"
+        destination = "/pkgs"
+      }
+
+      template {
+        data = file("xbps-clean-sigs")
+        destination = "local/xbps-clean-sigs"
+        perms = "0755"
+      }
+
+      template {
+        data = <<EOF
+[pkgs]
+path = /pkgs
+read only = no
+list = yes
+transfer logging = true
+timeout = 600
+exclude = - .* - *-repodata.* - *-stagedata.* - *.sig - *.sig2
```

Comment:
not 100% sure this is fully accurate and should it include the filters from the buildsync invocations? (`-f "+ */" -f "+ *-repodata" -f "+ *.xbps" -f "+ otime" -f "- *"`)